app/src/test/java/mozilla/lockbox/presenter/EditItemPresenterTest.kt

```diff
@@ -98,6 +102,11 @@ class EditItemPresenterTest {
         override fun setSaveEnabled(enabled: Boolean) {
             _saveEnabled = enabled
         }
+
+        override fun setTextSelectionOnPasswordToggle() {
+            // make sure the text selector is at the end of the line when password is toggled
```

Comment:
AH HA! That's what this does. Could you rename the method?

Reply:
😄